contentcuration/contentcuration/frontend/shared/views/FullscreenModal.vue

```diff
@@ -47,9 +47,9 @@
       <VContainer :style="`margin-top: ${contentOffset}px;`" fluid class="pa-0">
         <slot></slot>
       </VContainer>
-      <BottomToolBar v-if="$slots.bottom" color="white" flat clipped-left clipped-right>
+      <BottomBar v-if="$slots.bottom" color="white" flat clipped-left clipped-right>
```

Comment:
Here we also have a different view:

Before:
![image](https://github.com/learningequality/studio/assets/51239030/a7085849-d418-45e7-9e9a-3a2010b1e080)

![image](https://github.com/learningequality/studio/assets/51239030/bb080cf9-3290-4c7a-be0d-930a8a265c56)

After:
![image](https://github.com/learningequality/studio/assets/51239030/6523cff3-a994-4f6e-9755-e2f267beff50)

![image](https://github.com/learningequality/studio/assets/51239030/e8bb1cb6-96b8-4ab2-a284-b7011d07c3cb)